pages/glossary/disabled-element.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Disabled Element
+key: disabled-element
+---
+
+An element is considered disabled when it has been rendered [inoperable][] using one of the following properties:
```

Comment:
I don't think the definition of operable is not subjective (at least). With the definition of disabled being used in the applicability of the rule it needs to be.

Reply:
The means by which the element is rendered inoperable are however what's important and those are entirely unambiguous and objective. It's really just informative text that explains what the two properties accomplish, namely making an element inoperable.